rest-client-base/src/main/java/com/wultra/core/rest/client/base/RestClientConfiguration.java

```diff
@@ -496,4 +503,39 @@ public void setDefaultHttpHeaders(HttpHeaders headers) {
         this.defaultHttpHeaders = headers;
     }
 
+    /**
+     * Get the SSL handshake timeout.
+     *
+     * @return timeout duration
+     */
+    public Optional<Duration> getHandshakeTimeout() {
```

Comment:
Why do we use `Optional` for this parameter, but we do not use it for other parameters?

Reply:
I do not insist, it was just a suggestion how to do it. But yes, we ca be consistent.

Reply:
`Optional` removed.